token_dispenser/configuration.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+DEFAULT_LAUNCHPAD_GETTOKEN_URL:str =  "https://api.launchpad.nasa.gov/icam/api/sm/v1/gettoken"
+
+# Launchpad token api url
+LAUNCHPAD_GETTOKEN_URL: str = os.getenv('LAUNCHPAD_GETTOKEN_URL', DEFAULT_LAUNCHPAD_GETTOKEN_URL)
+# The secret-id point to the Launchpad pfx password
+LAUNCHPAD_PFX_PASSWORD_SECRET_ARN:str = os.getenv('LAUNCHPAD_PFX_PASSWORD_SECRET_ARN')
+# The bucket will launchpad.pfx is stored. Ex. my-sndbx-bucket
+LAUNCHPAD_PFX_FILE_S3_BUCKET = os.environ['LAUNCHPAD_PFX_FILE_S3_BUCKET']
+# The key to point to launchpad.pfx Ex. /folder1/folder2/launchpad.pfx
+LAUNCHPAD_PFX_FILE_S3_KEY = os.environ['LAUNCHPAD_PFX_FILE_S3_KEY']
+# DynamoDB is used for cache clientId and token. If not passed in , the program shall fail
+DYNAMO_DB_CACHE_TABLE_NAME: str = os.getenv('DYNAMO_DB_CACHE_TABLE_NAME')
+# The os.getenv('AWS_REGION') should get the lambda's running region
+AWS_REGION:str = os.getenv('AWS_REGION')
```

Comment:
Shouldn't we use brackets for all these since they're required for operation? We want `KeyError` if they don't exist at runtime